crates/interpreter/src/interpreter.rs

```diff
@@ -45,6 +45,8 @@ pub struct Interpreter<'a> {
     pub return_len: usize,
     /// Whether the interpreter is in "staticcall" mode, meaning no state changes can happen.
     pub is_static: bool,
+    /// EIP-3074: Active account for `AUTHCALL` instructions in the current execution frame.
+    pub active_account: Option<Address>,
```

Comment:
consider: rename to `authorized` for readability comparing to spec?

Reply:
Yeah good shout, will do 👍